package.json

```diff
@@ -92,6 +94,7 @@
 		"open-editor": "^1.0.1",
 		"path-exists": "^3.0.0",
 		"pkg-conf": "^2.0.0",
+		"prettier": "~1.8.2",
```

Comment:
@pvdlg The latest prettier version is currently `1.9.2`.

Reply:
Thanks. It's updated!